metricbeat/module/kubernetes/util/metrics_cache.go

```diff
@@ -0,0 +1,106 @@
+package util
+
+import (
+	"sync"
+	"time"
+)
+
+// PerfMetrics stores known metrics from Kubernetes nodes and containers
+var PerfMetrics *PerfMetricsCache
+
+const defaultTimeout = 120 * time.Second
```

Comment:
Should we make this configurable?

Reply:
It's a little tricky cause it's a global cache. I used a somehow safe value, I don't think someone needs to worry about it, what do you think?

Reply:
Let's do it until it breaks :-)